integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisManagementServiceIntegrationTest.java

```diff
@@ -136,6 +136,10 @@ public void tearDown() {
     client.cleanUp(authToken);
   }
 
+  private static String newRandomString(int length) {
+    return RandomStringUtils.insecure().next(length, true, true);
```

Comment:
Replacing something "deprecated" with something "insecure" does not look very nice to me :sweat_smile: 

Also (semi-related), using unseeded random input in tests could make them non-deterministic.

Do we really need random inputs here?

My impression is that we need to ensure that certain chars are permitted or not and the length is restricted. Using deterministic test parameters is probably a more natural approach here.

I understand that this PR merely attempts to avoid deprecation, but since the related test code came to focus here, WDYT about making a more substantial change to make the test deterministic and improve coverage of input cases?